lib/matplotlib/__init__.py

```diff
@@ -215,7 +222,76 @@ def _is_writable_dir(p):
     """
     return os.access(p, os.W_OK) and os.path.isdir(p)
 
+_verbose_msg = ("--verbose-LEVEL and the Verbose class are deprecated. "
+                "This functionality is now provided by the standard "
+                "python logging library.  To get more (or less) logging "
+                " output:\n"
+                "import logging\n "
+                "logger = logging.getLogger('matplotlib')."
+                "logger.set_level(logging.INFO)")
+
+
+def _set_logger_verbose_level(level_str='silent', file_str='sys.stdout'):
+    """
+    Use a --verbose-LEVEL level to set the logging level:
+
+    """
+    levelmap = {'silent': logging.WARNING, 'helpful': logging.INFO,
+                'debug': logging.DEBUG, 'debug-annoying': logging.DEBUG,
+                'info': logging.INFO, 'warning': logging.WARNING}
+    # Check that current state of logger isn't already more verbose
+    # than the requested level.  If its more verbose, then leave more
+    # verbose.
+    if level_str in levelmap:
```

Comment:
This check is actually performed in each case by upstream calling code, correct?

Reply:
Yes...  Well, it is now that I remembered to remove the call to this in `rc_params_from_file`